.gitignore

```diff
@@ -7,5 +7,4 @@ artifacts
 dist/test
 yarn-error.log
 local-docs
-bundle.js
-bundle.js.map
+./build
```

Comment:
As mentioned before, I think what we want to do is move the files from here to a temp dir after the compilation is completed, and not leave any artifacts in the working directory.

Having this git-ignored here is hiding the fact in our testing that when external developers compile/upload, it will leave git-visible artifacts in their project directory. They would need to know to git-ignore them manually.

If it's necessary to keep the build artifacts around after the CLI command has completed, we should move them to a temp dir and print the location, otherwise we should just delete them.

Reply:
hmm. I thought we agreed on to use a predefined output directory for the bundle outputs. 

well I think it's necessary to keep these build artifacts around so that the developer has a record of what was uploaded by the CLI. IMO in a typical workflow it's not moved into a temporary dir because that means it will get lost soon.

Reply:
oh I misunderstood your comment on the ticket but I disagree that it's a bad thing to leave temp build files in the working directory. I do feel like it's a common flow to allow ppl inspect / keep track of what's done. and that's why they need .gitignore.

Reply:
It's not that it's a problem to have artifacts in the working directory, it's just that I don't think the default behavior of the CLI should be to leave artifacts that need to be manually git-ignored by every single pack author. The reason I became aware that it was happening was because an alpha user already complained about it.

Reply:
alright. I hold my opinion but let's follow the user ask in this case.

Reply:
updated & tested.